frontend/src/globals.css

```diff
@@ -67,6 +67,8 @@
   --light-dark-background: #130713;
   --text-gray: #798795;
   --modal-border: #170f2e;
+  --warning-colour:rgba(255, 251, 0, 0.84);
+  --warning-colour-alt: rgb(70, 70, 1);
```

Comment:
colors do not match figma

Reply:
@whateverfw  how to get the color codes from figma ?

Reply:
in the right sidebar, just like the other stlye options

<img width="1728" alt="image" src="https://github.com/user-attachments/assets/622fbedb-493e-4095-9de5-367cfe9ffcfa" />

Reply:
Updated!